dockerfiles/cli/test.sh

```diff
@@ -31,7 +31,7 @@ fi
 #   The file has to be placed in tests folder in directory containing this script
 # (Optional) second argument is options for a docker run command.
 run_test_in_docker_container() {
-  docker_exec run --rm ${DOCKER_RUN_OPTIONS} $2 \
```

Comment:
why `$2` is removed this is used to pass some docker params like  in that test
`run_test_in_docker_container cmd_start_stop_restart_tests.bats --net=host`